Editors/vscode/src/extension.ts

```diff
@@ -29,7 +29,7 @@ export function activate(context: vscode.ExtensionContext) {
         synchronize: undefined
     };
 
-    const client = new langclient.LanguageClient('SourceKit Language Server', serverOptions, clientOptions);
+    const client = new langclient.LanguageClient('sourcekit-lsp', 'SourceKit Language Server', serverOptions, clientOptions);
```

Comment:
Assign 'sourcekit-lsp' to the id of LanguageClient because BaseLanguageClient use the value to get configuration.
In `vscode-languageclient/lib/client.js`:
```javascript
    refreshTrace(connection, sendNotification = false) {
        let config = vscode_1.workspace.getConfiguration(this._id);
        let trace = vscode_languageserver_protocol_1.Trace.Off;
        if (config) {
            trace = vscode_languageserver_protocol_1.Trace.fromString(config.get('trace.server', 'off'));
        }
        this._trace = trace;
        connection.trace(this._trace, this._tracer, sendNotification);
    }
```